site_python_floripa/conf.py

```diff
@@ -15,6 +15,7 @@
 NAVIGATION_LINKS = {
     DEFAULT_LANG: (
         ('/', 'Início'),
+        ('https://floripa.sc.python.org.br/v1/', 'Site Original'),
```

Comment:
Sugestão:
```suggestion
        ('https://floripa.sc.python.org.br/v1/comunidade/', 'Comunidade'),
        ('https://floripa.sc.python.org.br/v1/encontros/', 'Encontros'),
        ( 'https://floripa.sc.python.org.br/v1/empresas/', 'Empresas'),
```
Dessa forma, a pessoa já vai direto para as páginas.

Reply:
Acho que acaba ficando um pouco confuso, já que clicamos em uma seção do site novo e caímos em uma seção do original. Minha sugestão seria deixar o link para o site original apenas (que ficaria ali no cabeçalho mesmo quando o site novo for finalizado), como está agora. 

Tem a parte ruim de não ter uma seção "contato" (com links das redes sociais da comunidade), por exemplo, no site novo hoje. Mas deixando ele o mais cru possível, pode ser que atraia novas pessoas (iniciantes/júnior) para ajudar com comunidade (de inicio com o site, o qual deixamos um guia de como fazer na [wiki](https://github.com/PythonFloripa/pythonfloripa.github.io/wiki/Site-Python-Floripa-2021) e algumas ideias de páginas para se contribuir aqui no [projeto](https://github.com/PythonFloripa/pythonfloripa.github.io/projects/1) e vamos ficar sempre chamando o pessoal a contribuir nos eventos online!).

O que acham?

Reply:
Concordo com o @mateusoliveira43 acho que fica confuso conforme a sugestão do @rougeth. Seria melhor colocar diretamente, como foi feito no PR.